nipoppy/workflows/pipeline.py

```diff
@@ -784,11 +787,11 @@ def _submit_hpc_job(self, participants_sessions):
 
         try:
             qa.switch_cluster(self.hpc)
-        except KeyError:
-            raise ValueError(
+        except KeyError as e:
+            raise ConfigError(
```

Comment:
This is from the `--hpc` flag, not a config file

```suggestion
            raise WorkflowError(
```